config/webpack/webpack.config.js

```diff
@@ -1,11 +1,5 @@
-// use the new NPM package name, `shakapacker`.
-// merge is webpack-merge from https://github.com/survivejs/webpack-merge
-const { webpackConfig: baseWebpackConfig, merge } = require('shakapacker')
+const { generateWebpackConfig } = require('shakapacker')
 
-const options = {
```

Comment:
is it safe to remove these options?

Reply:
They did not break any current implementations in my test so far but I'll add them back